go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/burnt-labs/xion
 
-go 1.19
+go 1.20
 
 require (
-	github.com/CosmWasm/wasmvm v1.4.1 // indirect
+	github.com/CosmWasm/wasmvm v1.5.0 // indirect
```

Comment:
make sure to use 1.5.2 since 1.5 has a mem leak and high severity security issue